CHANGELOG.md

```diff
@@ -21,8 +21,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Move `eslint` to devDependencies ([#489][] @gmfvpereira)
+- Display wrong numbers in Person Contact Kit ([#499][] @KFad11)
```

Comment:
Also, can we be a bit more descriptive? The Ninjas will look at this to figure out what to test. What should they look for?